tests/e2e/lib/virt_helpers.go

```diff
@@ -601,58 +602,6 @@ func (v *VirtOperator) checkVmStatus(namespace, name, expectedStatus string) boo
 	return status == expectedStatus
 }
 
-func (v *VirtOperator) createVm(namespace, name, source string) error {
```

Comment:
the disk in code yaml can also be deleted?

Reply:
Do you mean the DataVolume creation code in createDataVolumeFromSource? It is still used for downloading the CirrOS image. I haven't found a good way to combine fixed YAML with the code that gets the latest CirrOS version automatically.

Reply:
I see, disk yaml is only from pvc part. still missing a from url part, right?

Reply:
since it is optimization, I would just create an issue and continue this PR (when is ready to merge, I suggest rebasing with master, it gets merged way quicker)

Reply:
Yes, I didn't add a YAML for a disk with a URL source, because the CirrOS URL can change. We could go back to updating it manually if it would be better to have that in YAML too. But right now there's a getLatestCirrosImageURL that checks for the latest URL, and I don't see a nice way to get the resulting URL into YAML without adding a bunch of code for templating or string substitution or something like that.